apps/api/src/modules/user/user.service.spec.ts

```diff
@@ -1,13 +1,18 @@
+import { ConfigModule, ConfigService } from '@nestjs/config'
+import { JwtModule } from '@nestjs/jwt'
 import { Test, TestingModule } from '@nestjs/testing'
+import { PrismaService } from 'src/database/prisma.service'
 
+import { AuthModule } from '../auth/auth.module'
+import { AuthRepository } from '../auth/auth.repository'
 import { UserService } from './user.service'
 
 describe('UserService', () => {
   let service: UserService
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [UserService],
+      providers: [UserService, AuthRepository, PrismaService],
     }).compile()
 
```

Comment:
should we write unit test for `getUserData`

Reply:
Later